packages/rich-text/src/plugins/List/components/List.tsx

```diff
@@ -8,10 +8,15 @@ import * as Slate from 'slate-react';
 const baseStyle = css`
   padding: 0;
   margin: 0 0 1.25rem 1.25rem;
+
   div:first-child {
     margin: 0;
     line-height: ${tokens.lineHeightDefault};
   }
+
+  li + li {
+    margin-top: ${tokens.spacingXs};
```

Comment:
What's the motivation for this one?

I actually noticed some odd spacing behavior playing around with this:
![spacing-issues](https://user-images.githubusercontent.com/101926/150831062-e74ce9bc-b028-474c-a95c-23060b7f30c9.gif)

Reply:
Good catch. It was supposed to provide consistent spacing between li elements. I took it from the old editor code but looks like I broke something else. will check 👍🏻

Reply:
pushed a fix